plugins/node/opentelemetry-instrumentation-restify/package.json

```diff
@@ -54,7 +54,7 @@
     "gts": "3.1.0",
     "mocha": "7.2.0",
     "nyc": "15.1.0",
-    "restify": "4.3.4",
+    "restify": "8.6.1",
```

Comment:
Thank you for submitting a fix. I think the original error originated from [`@types/restify` here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/ec9ee131635dc2db88deea4f2efb887ff6f60577/plugins/node/opentelemetry-instrumentation-restify/package.json#L66) which is a regular dependency. Your PR addresses the dev dependency on "restify" packages, which might be updated as well regardless IMO.

This seems reasonable to me, as version `4.3.4` is 4 years old and has only 20K downloads vs 313K for most popular version.

I wonder if the reason that `4.3.4` produced an error implies that the instrumentation no longer supports it's [advertised range of >= 4.0.0](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-restify#supported-versions)? (probably due to some other dependency that broke types?)

This instrumentation has no `test-all-versions` setup in place, so it's difficult to tell, but maybe you investigated this issue and can share insights?